build/common.gni

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains common build flags for building Crosswalk. Users should
+# NOT directly import this file into their build folder.
+# See //xwalk/build/linux.gni for how to import flags for Crosswalk-Linux Build
+
+# Crosswalk specific build flags are defined in xwalk.gni. Import them.
+import("//xwalk/build/xwalk.gni")
+
+root_extra_deps = [ "//xwalk:xwalk_builder" ]
+
+# From src/build/config/features.gni
+enable_nacl = true
+
+# From src/build/config/features.gni
```

Comment:
Its **very** important not to discard the comments from the gyp version. If I just look at this assignment I have no idea why it is being made.